assets/src/dashboard/components/cardGridItem/cardPreview.js

```diff
@@ -154,6 +155,14 @@ const CardPreviewContainer = ({
     return () => intervalId && clearInterval(intervalId);
   }, [storyPages.length, cardState]);
 
+  const handleKeyDownEditControls = useCallback(
+    ({ nativeEvent }) => {
+      if (nativeEvent.keyCode === 13) {
```

Comment:
Nit: Could you use the `.key` (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) property instead?   It makes things much more readable and easy to understand what key we're actually looking for